fe/fe-core/src/main/java/com/starrocks/qe/StmtExecutor.java

```diff
@@ -439,6 +447,18 @@ public void execute() throws Exception {
                             parsedStmt = selectStmt;
                             execPlan = StatementPlanner.plan(parsedStmt, context);
                         }
+                    } else if (parsedStmt instanceof ExecuteStmt) {
+                        ExecuteStmt executeStmt = (ExecuteStmt) parsedStmt;
+                        com.starrocks.sql.analyzer.Analyzer.analyze(executeStmt, context);
+                        prepareStmtContext = context.getPreparedStmt(executeStmt.getStmtName());
+                        if (null == prepareStmtContext) {
+                            throw new StarRocksPlannerException(ErrorType.INTERNAL_ERROR,
+                                    "prepare statement can't be found @ %s, maybe has expired", executeStmt.getStmtName());
+                        }
+                        PrepareStmt prepareStmt = prepareStmtContext.getStmt();
+                        parsedStmt = prepareStmt.assignValues(executeStmt.getParamsExpr());
+                        parsedStmt.setOrigStmt(originStmt);
+                        execPlan = StatementPlanner.plan(parsedStmt, context);
```

Comment:
Have parameters had been replaced by these values in the parsedStmt? Could you explain how you did this?

Reply:
AstBuilder :  visitExecuteStatement. this process is dirty, normally use visit all parse nodes, but node don't have this this abstract. so i use context pass exprs